llvm/lib/Target/AMDGPU/SIInstrInfo.td

```diff
@@ -1129,6 +1129,8 @@ def exp_tgt : CustomOperand<i32, 0, "ExpTgt">;
 
 def wait_vdst : NamedIntOperand<i8, "wait_vdst", "WaitVDST">;
 def wait_exp : NamedIntOperand<i8, "wait_exp", "WaitEXP">;
+def WaitVAVDst : NamedIntOperand<i8, "wait_va_vdst">;
```

Comment:
For what it's worth, I think the definitions here should probably match the SP3 names, so `wait_va_vdst` and `wait_vm_vsrc`.